packages/agreement-process/.env

```diff
@@ -17,6 +17,7 @@ READMODEL_DB_PASSWORD=example
 READMODEL_DB_PORT=27017
 
 WELL_KNOWN_URLS="https://dev.interop.pagopa.it/.well-known/jwks.json"
+ACCEPTED_AUDIENCES="dev.interop.pagopa.it/ui, refactor.dev.interop.pagopa.it/ui"
```

Comment:
does zod trim the strings after split?
it's ok if it doesn't, in that case it's better to remove the space in `.env` files

Reply:
I agree with @galales.
@ecamellini, could you remove that space?
Thanks.